<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Текстовый редактор</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <widget class="QPushButton" name="newButton">
                                <property name="text">
                                    <string>Новый</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="openButton">
                                <property name="text">
                                    <string>Открыть</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="saveButton">
                                <property name="text">
                                    <string>Сохранить</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="horizontalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QLabel" name="fileNameLabel">
                                <property name="text">
                                    <string>Файл не выбран</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <widget class="QTextEdit" name="textEdit">
                        <property name="html">
                            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
                                &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style
                                type=&quot;text/css&quot;&gt;
                                p, li { white-space: pre-wrap; }
                                &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2';
                                font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
                                &lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px;
                                margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br
                                /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                            </string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QStatusBar" name="statusbar"/>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>21</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuФайл">
                <property name="title">
                    <string>Файл</string>
                </property>
                <addaction name="actionNew"/>
                <addaction name="actionOpen"/>
                <addaction name="actionSave"/>
                <addaction name="separator"/>
                <addaction name="actionExit"/>
            </widget>
            <addaction name="menuФайл"/>
        </widget>
        <action name="actionNew">
            <property name="text">
                <string>Новый</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+N</string>
            </property>
        </action>
        <action name="actionOpen">
            <property name="text">
                <string>Открыть</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+O</string>
            </property>
        </action>
        <action name="actionSave">
            <property name="text">
                <string>Сохранить</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+S</string>
            </property>
        </action>
        <action name="actionExit">
            <property name="text">
                <string>Выход</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+Q</string>
            </property>
        </action>
    </widget>
    <resources/>
    <connections/>
</ui>